### PR TITLE
Conferences: Section of 'Personnel'

### PR DESCRIPTION
**Pull request by martha-thomae**
Here is the section of 'personnel' for each conference. The section now includes the local organizing committee, the program committee, and the keynotes. I changed its naming now that keynotes are included here too.

All the information about the staff involved is in the header of the file and it is loaded into the body using the template `_layouts/people.html`

I tried to make each entry consistent, always following the structure:
- name
- chair (optional)
- institutions
- country

This looks odd in two entries: "RISM Switzerland, Switzerland" and "National Library of Norway, Norway" (but only for those two).

One remaining inconsistency would be that in some years, there is no indication about who is the chair in a given committee (this is true for the oldest conferences). And another is the ordering of the names, in some years the order seems to be random, but with the chairs always at the beginning; in other years, all entries are ordered alphabetically. Any preferences to fix this?

### CODE REVIEW

**Comment by musicEnfanthen** [comment]
Thanks @martha-thomae , looks great.

At least for "RISM Switzerland, Switzerland", you could go for "RISM CH, Switzerland", I guess (@lpugin ?)
@margrethesb Are you ok with "National Library of Norway, Norway"?

Would it make sense to have the file names, permalinks and layout to say "personnel" instead of "people", too?

Ad chairs: The only ones without chairs I can see are the OC's 2013 and 2014. I guess that's because everyone named there was co-chair at that time? (@kepper )

File: _layouts/people.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: "People Involved in the Conference"
+---
+
+<div style="margin-top:2em;">
+    <h2>Local Organizing Committee</h4>
+    <ul>
+    {% for omember in page.organizers %}
+        <li><span style="font-weight: 400;">{{ omember }}</span></li>
```
Comment (musicEnfanthen) [inline _layouts/people.html:10]: I am not sure that `<span style="font-weight: 400;">` is necessary. Does it change anything when you remove it? (see also below)

File: _layouts/people.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    {% endfor %}
+    </ul>
+    
+    <h2>Program Committee</h4>
+    <ul>
+    {% for pmember in page.program %}
+        <li><span style="font-weight: 400;">{{ pmember }}</span></li>
```
Comment (musicEnfanthen) [inline _layouts/people.html:17]: see above

File: _layouts/people.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    {% endfor %}
+    </ul>
+
+    <h2>Keynote Speakers</h4>
+    <ul>
+    {% for speaker in page.keynotes %}
+        <li><span style="font-weight: 400;">{{ speaker }}</span></li>
```
Comment (musicEnfanthen) [inline _layouts/people.html:24]: see above

File: _layouts/people.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: "People Involved in the Conference"
+---
+
+<div style="margin-top:2em;">
+    <h2>Local Organizing Committee</h4>
```
Comment (musicEnfanthen) [inline _layouts/people.html:7]:
```suggestion
    <h2>Local Organizing Committee</h2>
```
Should be h2 or h4

File: _layouts/people.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    <ul>
+    {% for omember in page.organizers %}
+        <li><span style="font-weight: 400;">{{ omember }}</span></li>
+    {% endfor %}
+    </ul>
+    
+    <h2>Program Committee</h4>
```
Comment (musicEnfanthen) [inline _layouts/people.html:14]:
```suggestion
    <h2>Program Committee</h2>
```

File: _layouts/people.html [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    <ul>
+    {% for pmember in page.program %}
+        <li><span style="font-weight: 400;">{{ pmember }}</span></li>
+    {% endfor %}
+    </ul>
+
+    <h2>Keynote Speakers</h4>
```
Comment (musicEnfanthen) [inline _layouts/people.html:21]:
```suggestion
    <h2>Keynote Speakers</h2>
```

File: _layouts/people.html [hunk 7]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: "People Involved in the Conference"
+---
+
+<div style="margin-top:2em;">
```
Comment (musicEnfanthen) [inline _layouts/people.html:6]:
```suggestion
<div class="columns">
  <div class="column col-10 col-sm-12">
```
Could you please try this, for responsive design. Will need an additional closing `</div>` at the end.

File: _conferences/2020/people.html [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+    - "Jessica Fulkerson, Lecturer in Music, Tufts University, USA"
+program:
+    - "Richard Freedman (Committee Chair), Haverford College, USA"
+    - "Vincent Besson, Centre d'études supérieures de la Renaissance, Université de Tours, France"
+    - "Margrethe Bue, National Library of Norway, Norway"
+    - "Joy Calico, Vanderbilt University, USA"
+    - "Elsa De Luca, NOVA University of Lisbon, Portugal"
+    - "Stefan Münnich, University of Basel, Switzerland"
+    - "Anna Plaksin, Max Weber Stiftung, Germany"
+    - "David Weigl, University of Music and Performing Arts Vienna, Austria"
```
Comment (musicEnfanthen) [inline _conferences/2020/people.html:21]:
```suggestion
    - "David M. Weigl, University of Music and Performing Arts Vienna, Austria"
```
Sort by M.

File: _conferences/2019/people.html [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+    - "Johanna Devaney, Brooklyn College, CUNY, USA"
+    - "Andrew Hankinson, Bodleian Libraries, UK"
+    - "Anna E. Kijas, Boston College, USA"
+    - "Kevin R. Page (Committee Chair), University of Oxford, UK"
+    - "Klaus Rettinghaus, Saxon Academy of Sciences and Humanities in Leipzig, Germany"
+    - "Agnes Seipelt, University of Paderborn, Germany"
+    - "Raffaele Viglianti, University of Maryland, USA"
+keynotes:
+    - "Georg Vogeler, Graz, Austria"
+    - "Johanne Devaney, Brooklyn College, CUNY, USA"
```
Comment (musicEnfanthen) [inline _conferences/2019/people.html:24]:
```suggestion
    - "Johanna Devaney, Brooklyn College, CUNY, USA"
```

File: _conferences/2020/people.html [hunk 10]
```diff
@@ -0,0 +1,25 @@
+---
+layout: people
+title: "Personnel"
+permalink: "/conference/2020/people/"
+tag: MEC2020
+id: personnel
+organizers:
+    - "Anna Kijas (Committee Chair), Head, Lilly Music Library, Tufts University, USA"
```
Comment (musicEnfanthen) [inline _conferences/2020/people.html:8]:
```suggestion
    - "Anna E. Kijas (Committee Chair), Head, Lilly Music Library, Tufts University, USA"
```

File: _conferences/2018/people.html [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+    - "Raffaele Viglianti (Committee Co-Chair), Maryland Institute for Technology in the Humanities, USA"
+    - "Leighann Yarwood, The Clarice Smith Performing Arts Center, USA"
+program:
+    - "Karen Desmond (Committee Chair), Brandeis University, USA"
+    - "Johanna Devaney, Ohio State University, USA"
+    - "David Fiala, Centre d’études supérieures de la Renaissance, Université de Tours, France"
+    - "Andrew Hankinson, Bodleian Libraries, University of Oxford, UK"
+    - "Maja Hartwig, University of Paderborn, Germany"
+keynotes:
+    - "Anna Kijas, Boston College, USA"
```
Comment (musicEnfanthen) [inline _conferences/2018/people.html:19]:
```suggestion
    - "Anna E. Kijas, Boston College, USA"
```

File: _conferences/2018/people.html [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+id: personnel
+organizers:
+    - "Amanda Lee-Barber, The Clarice Smith Performing Arts Center, USA"
+    - "Stephen Henry (Committee Co-Chair), Michelle Smith Performing Arts Library, USA"
+    - "Raffaele Viglianti (Committee Co-Chair), Maryland Institute for Technology in the Humanities, USA"
+    - "Leighann Yarwood, The Clarice Smith Performing Arts Center, USA"
+program:
+    - "Karen Desmond (Committee Chair), Brandeis University, USA"
+    - "Johanna Devaney, Ohio State University, USA"
+    - "David Fiala, Centre d’études supérieures de la Renaissance, Université de Tours, France"
```
Comment (musicEnfanthen) [inline _conferences/2018/people.html:15]:
```suggestion
    - "David Fiala, Centre d’Études Supérieures de la Renaissance, Université de Tours, France"
```

File: _conferences/2017/people.html [hunk 13]
```diff
@@ -0,0 +1,23 @@
+---
+layout: people
+title: "Personnel"
+permalink: "/conference/2017/people/"
+tag: MEC2017
+id: personnel
+organizers:
+    - "Philippe Vendrix (Committee Co-Chair), Centre d’études supérieures de la Renaissance (CESR), Université de Tours, France"
```
Comment (musicEnfanthen) [inline _conferences/2017/people.html:8]:
```suggestion
    - "Philippe Vendrix (Committee Co-Chair), Centre d’Études Supérieures de la Renaissance (CESR), Université de Tours, France"
```

File: _conferences/2017/people.html [hunk 14]
```diff
@@ -0,0 +1,23 @@
...
+    - "Philippe Rigaux (Committee Co-Chair), Conservatoire national des arts et métiers (CNAM), France"
```
Comment (musicEnfanthen) [inline _conferences/2017/people.html:9]:
```suggestion
    - "Philippe Rigaux (Committee Co-Chair), Conservatoire National des Arts et Métiers (CNAM), France"
```

**Review by musicEnfanthen — CHANGES_REQUESTED** [review]
Thanks @martha-thomae . Since no one else responded, I'd suggest the following to go on with this PR:

Order of entries: Choose one order system (alphabetically with chairs included, OR alphabetically with chairs on top) and use it for all years. I'd prefer all alphabetical style, but the chairs on top version is sometimes easier to read. I'll leave that decision to you.

For years when there is no chair, there is none, that's fine.

Organization, Country: "National Library of Norway, Norway" should be fine. The first one is the name of the organization, the second one the country. Same for RISM, although we could change to "RISM CH, Switzerland", here.

Looking at the naming again, we should make it consistent, too. So when we stick with  "people", the ids should also be "people". Thus only the title will be "Personnel". (I have been thinking about "roles and responsibilities" lately, , but let's leave this for another PR).

Does the `people.html` show up properly for each year on your branch? Can't test it here.

**Comment by bwbohl** [comment]
@musicEnfanthen thanks for pushing this forward and your great remarks, and @martha-thomae of course thanks for the initial impetus!
I think you both are n top of this and can decide how you see best!

**Comment by martha-thomae** [comment]
Thanks, @musicEnfanthen. I will move on with your changes. Thanks for the reminder, @bwbohl.

File: _layouts/people.html [hunk 15]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
```
Comment (musicEnfanthen) [inline _layouts/people.html:2]:
```suggestion
layout: conference
```
This should be `conference`, to keep the side bar menu displayed.

**Comment by martha-thomae** [comment]
@musicEnfanthen, thanks for all the suggestions, everything should be fine now!
And thanks for the patience, the last months of this year have been a bit hard for personal reasons. But now back to normality.

> Does the `people.html` show up properly for each year on your branch? Can't test it here.

Yes, all of them do show :)

**Review by musicEnfanthen — APPROVED** [review]

**Comment by musicEnfanthen** [comment]
Thanks @martha-thomae for your work, looks great!

**Pull request merged**